db/plain_table_db_test.cc

```diff
@@ -547,6 +555,50 @@ TEST_P(PlainTableDBTest, Flush2) {
   }
 }
 
+TEST_P(PlainTableDBTest, Immortable) {
```

Comment:
Is this a portmanteau of immortal+immutable? I see only one other use of this word in the codebase.

Reply:
Good point:) It's immortal...

Reply:
I liked the neologism though :)